### ISSUE TITLE
movie_strip_add sometimes offset audio strips by a few frames in 2.78

### ISSUE THREAD

**Issue by NathanLovato** [GDquest/Blender-power-sequencer#57]
I've got no idea why this happens, but calling `movie_strip_add` sometimes offsets all audio strips in the current scene by 5 frames. Right now it seems like a Blender internal bug, not sure.

- [x] Test in Blender 2.79

**Comment by NathanLovato** [comment]
It was way worse in 2.79, but found the fix: set the `use_framerate` argument to False / even if all videos were the same framerate, it wouldn't change the project's framerate but break the audio strips' framerate somehow.

**Comment by NathanLovato** [comment]
Fixed and found the source of the issue at last: the `mkv` container. Now I get why this came out of nowhere: I started recording in .mkv because even if the program crashes in the middle of something, the recording's not lost (unlike with .mp4). But Blender seems to struggle when importing these. And if in 2.79 it's clearly visible in 2.78 it wasn't.

The thing is: it only happens upon calling `bpy.ops.sequencer.movie_strip_add` - it doesn't happen if you use the add menu. Anyway, closing this.
